test/cpp/src/750-channel-dimensions.cc

```diff
@@ -1,3 +1,5 @@
+
+#ifndef RAMULATOR_TEST
```

Comment:
We could potentially make it work, but the test itself will need to change a lot. Specifically, we will need to instantiate a MEMORY_CONTROLLER instead of individual channels. Ramulator simulates from the memory controller level, so there is one instance of Ramulator and it runs from the memory controller instead of the channels.